engine/src/main/java/com/redhat/cloud/notifications/recipients/recipientsresolver/RecipientsResolverService.java

```diff
@@ -18,5 +19,6 @@ public interface RecipientsResolverService {
     @Path("/internal/recipients-resolver")
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
+    @Retry
```

Comment:
Because the only usage of this method is already on handled by a retry policy, why adding `@Retry` here  ?

Reply:
Good catch! I forgot we had the retry policy from FailSafe.

Reply:
Removed.